spec/models/alert_feed_spec.rb

```diff
@@ -0,0 +1,52 @@
+require 'rails_helper'
+
+RSpec.describe AlertFeed, type: :model do
+  before :each do
+    @region = Region.create!(
```

Comment:
Not saying you have to change anything; this is strictly for my personal edification. Why did you choose to create an instance variable here instead of using `let`?